CIP-0105/test-vectors/test-vector-1.md

```diff
@@ -42,19 +42,19 @@ Hex: `f74d7ac30513ac1825715fd0196769761fca6e7f69de33d04ef09a0c417a752b1d84110299
 
 Bech32: `drep_xvk17axh4sc9zwkpsft3tlgpjemfwc0u5mnld80r85zw7zdqcst6w543mpq3q2vkjy3nw8x7n8asw4es78dyl4q7u7kwlwn7yy0sugxfrjs6z25qe`
 
-#### Verification key hash (DRep ID)
+#### Verification key hash (DRep VKH)
```

Comment:
```suggestion
#### [Deprecated] Verification key hash (DRep ID)
```

might be worth keeping the test vectors for the deprecated IDs as well as adding new ones

Reply:
done